.github/workflows/end_to_end_test.yaml

```diff
@@ -0,0 +1,61 @@
+name: End to end tests
+
+on:
+  pull_request:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  e2e-test:
+    name: end to end test
+    runs-on: [self-hosted, linux, x64, e2e-runner]
+    steps:
+      - name: Echo hello world
+        run: echo "hello world"
+      - name: File permission for /usr/local/bin
+        run: ls -ld /usr/local/bin | grep drwxrwxrwx
+      - name: Test file permission for /usr/local/bin
+        run: touch /usr/local/bin/test_file
+      # "Install microk8s" step will test if the proxies settings are correct.
+      - name: Proxy set in /etc/environment
+        run: cat /etc/environment | grep HTTP_PROXY
```

Comment:
What's the purpose of this `cat /etc/environment | grep HTTP_PROXY `?

Reply:
Minor comments